index.html

```diff
@@ -80,7 +80,8 @@ <h1 id="title"><i class="todo">PROPOSED</i> Decentralized Identifier Working Gro
 <a href="https://www.w3.org/2019/did-wg/">Decentralized Identifier Working Group</a>
 is to maintain the <a href="https://www.w3.org/TR/did-core/">Decentralized Identifiers (DIDs)</a>
 specification and related Working Group Notes. The WG will also seek consensus
-around the specification of DID Methods and DID Resolution.
+around the best way to achieve effective interoperability, possibly through the
+specification of DID Resolution and/or DID methods.
```

Comment:
This language seems correct to me, based on the experience of managing the did spec registries, and having written many of the did core test suites.